docs/relay/relay.md

```diff
@@ -11,23 +11,42 @@ Many factors impact how you connect to the remote host, and it can be tricky to
 
-Netcode doesn't offer tools to help you do a successful punch through a NAT. But, Unity Services provides a Relay service that can relay all Unity Transport based technology, like Netcode.
+Netcode doesn't offer tools to help you do a successful punch through a NAT. However, Unity Services provides a Relay service that can relay all Unity Transport based technology, like Netcode.
 
 # How does Relay work?
 
-You need to install the Unity Relay SDK if you want to interact with the service. To install it, add `com.unity.services.relay` package to your project.
-
-The host of your game needs to request a Relay allocation first. This process reserves capacity on a Relay server that allows the clients to bind to it. After you request an allocation, you can get a [join code](https://docs.unity.com/relay/en/manual/join-codes) for the allocation. The join code is a random string that your clients will offer to the SDK to join the correct server and allocation.
+The host of your game will need to request a Relay allocation and get a [join code](https://docs.unity.com/relay/en/manual/join-codes) for this allocation. The join code is a random string that your clients will use to join the correct server and allocation. The host and clients will then be able to communicate via the Relay server without disclosing their IP addresses and ports directly.
 
 ## How do I enable Relay for my project?
 
-The first thing to understand is that Relay is a Unity service. You need to add it to your organization in the Unity Dashboard (under the **Multiplayer** section). After that, link your project to your organization in the Unity Editor (**Project Settings** > **Services**).
+You need to install the Unity Relay SDK if you want to interact with the service. To install it, add `com.unity.services.relay` package to your project.
+
+You need to add the Relay Service to your organization in the Unity Dashboard (under the **Multiplayer** section). After that, link your project to your organization in the Unity Editor (**Project Settings** > **Services**).
 
 See [Get started with Relay](https://docs.unity.com/relay/en/manual/get-started) for more detailed instructions.
 
+## How do I try Relay in the Editor?
+
+If you are using Unity 2022.3 or later, you can try the Relay service directly in the editor from the inspector of the Network Manager. The workflow is similar to what you experienced in
+the [Get Started with NGO](../tutorials/get-started-with-ngo.md) page: the inspector shows buttons which start a Host, Server or Client without the need to add a UI to your game yet. The biggest difference is that starting a client requires a [join code](https://docs.unity.com/relay/en/manual/join-codes) which you can get from the host or server.
```

Comment:
Do we explain how to get that join code from the host or server?

Reply:
Hmm it is in the snippet below (GetJoinCode). Rephrase it a bit if you think it is not clear